src/server/imageProcessing/process.py

```diff
@@ -8,7 +8,7 @@
 
 
 pytesseract.pytesseract.tesseract_cmd = (
-    r"C:\Program Files\Tesseract-OCR\tesseract.exe"
+    "/usr/local/bin/tesseract"
```

Comment:
i feel like this path may be different for each person depending on where we set up our env vars and if we’re using a mac OS/ linux/ windows, maybe we can create a local .env file for now and set our own paths?

Reply:
yup exactly, probs can do this after the demo